app/timetables/migrations/0001_initial.py

```diff
@@ -14,10 +15,38 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.CreateModel(
+            name='Meal',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=60, unique=True)),
+                ('start_time', models.TimeField()),
+                ('end_time', models.TimeField()),
+            ],
+            bases=(common.mixins.ForceCapitalizeMixin, models.Model),
+        ),
+        migrations.CreateModel(
+            name='Timetable',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('date_created', models.DateTimeField(auto_now_add=True)),
+                ('date_modified', models.DateTimeField(auto_now=True)),
+                ('name', models.CharField(max_length=60, unique=True)),
+                ('code', models.CharField(max_length=60, unique=True)),
+                ('api_key', models.CharField(max_length=60, unique=True)),
+                ('cycle_length', models.IntegerField()),
+                ('current_cycle_day', models.IntegerField()),
+            ],
+            options={
+                'abstract': False,
+            },
+            bases=(common.mixins.ForceCapitalizeMixin, models.Model),
+        ),
+        migrations.CreateModel(
             name='Weekday',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('name', models.CharField(max_length=60, unique=True)),
             ],
+            bases=(common.mixins.ForceCapitalizeMixin, models.Model),
         ),
     ]
```

Comment:
Meal model already has it's own migration file. You should not have mixed it up in initial.py file.